releases/release-1.31/release-notes/maps/pr-124383-map.yaml

```diff
@@ -0,0 +1,6 @@
+pr: 124383
+releasenote:
+  text: |-
+    kube-proxy's nftables mode (--proxy-mode=nftables) is now beta and available by default.
+
+    FIXME ADD MORE HERE BEFORE THE RELEASE, DOCS LINKS AND STUFF
```

Comment:
This is already tracked in the sheet to reach out to authors at the end of the release.

Reply:
So should i delete this one?

Reply:
We can remove the `FIXME ADD MORE HERE BEFORE THE RELEASE, DOCS LINKS AND STUFF` section to clean it up. We'll reach out to the authors before the final published notes to make sure this looks correct and has links to docs.